docs/release-notes/2.38.0.md

```diff
@@ -0,0 +1,236 @@
+- [PMM-12038](https://jira.percona.com/browse/PMM-12038) - [Backup]: Extended support for MongoDB backups to cover all authentication mechanisms available for MongoDB (including LDAP, x509 certs).
+- [PMM-9544](https://jira.percona.com/browse/PMM-9544) -   [Alerting]: Added new default alert template to Node status and notify if the Agent is down.
+- [PMM-12114](https://jira.percona.com/browse/PMM-12114) - Upgraded Grafana to 9.2.18 to fix to CVEs.
+- [PMM-12136](https://jira.percona.com/browse/PMM-12136) - We have migrated our base operating system on which we build our base image from CentOS 7 to Oracle Linux 9 to fix the CVEs.
+
+## Bugs Fixed
+
+- [PMM-11578](https://jira.percona.com/browse/PMM-11578) - [Alerting]: The **Last notified** column under **Alerting > Fired Alerts** has been renamed to **Last triggered**. This better illustrates the fact that triggered alerts may not always generate a notification. This can happen when the contact points are not correctly configured.
+In addition, the **Suppressed** status in the **State** column has been renamed to **Silenced** to keep consistency with how this status is displayed on other **Alerting** pages.
+- [PMM-12067](https://jira.percona.com/browse/PMM-12067) - [QAN]: Fixed vulnerability in Vitess package.
+- [PMM-12105](https://jira.percona.com/browse/PMM-12105) - [QAN]: Fixed an issue where QAN was broken due to wrong pgsm_query_ids.
+- [PMM-10063](https://jira.percona.com/browse/PMM-10063) - [QAN]: The QAN filter section could display the available list of services/nodes. But the load time was too much, with the UI  being unresponsive. This issue has been resolved now. The entire list is loaded with the count displayed on the upper right side.
+- [PMM-11386](https://jira.percona.com/browse/PMM-11386) - [QAN]: MongoDB QAN query time metrics was showing the wrong unit. It should be ms. The issue has been resolved now.
+- [PMM-12024](https://jira.percona.com/browse/PMM-12024) - [QAN]: Fixed an issue where a QAN agent couldn't start due to an expected return value type.
+- [PMM-12091](https://jira.percona.com/browse/PMM-12091) - [QAN]: The tooltip for QAN > MongoDB > Query Details was displayed in seconds instead of decimals. The issue has been fixed now.
+- [PMM-9844](https://jira.percona.com/browse/PMM-9844) -   [QAN]: Data for a large number of partitions or possible indexes was shown in the QAN Details tab in one scrolling column making it practically impossible to read. The issue has been resolved now.
+- [PMM-11938](https://jira.percona.com/browse/PMM-11938) - [QAN]: Fixed the problem when PMM tried to create `pg_stat_monitor_settings` view all over again.
+- [PMM-12031](https://jira.percona.com/browse/PMM-12031) - [Inventory]: The `enhanced_metrics_disabled` and `basic_metrics_disabled` were not displayed on the **Inventory** page for an external service. The issue has been resolved now.
```

Comment:
@atymchuk I think we can improve the description for this ticket.

Reply:
removed it considering Alex's feedback